compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp

```diff
@@ -499,6 +499,24 @@ class ConvertNCHWToNHWC final : public luci::CircleNodeMutableVisitor<bool>
 
     return true;
   }
+
+  bool visit(luci::CircleRelu6 *node)
+  {
+    const auto pred_node = loco::must_cast<luci::CircleNode *>(node->features());
+    auto pre_trans = create_pre_transpose(node);
+    pre_trans->a(pred_node);
+    node->features(pre_trans);
+
+    // Make loco do shape inference for this node again.
```

Comment:
```suggestion
    // Do shape inference for this node again.
```

Reply:
_loco_ does not do shape inference, _luci_ does...

Reply:
I see. I've updated the comment. Thanks.

Reply:
> I've updated the comment (for other nodes as well). Thanks.

Please update for other nodes in another PR. This PR is about RELU6.

Reply:
@jinevening FYI, `shape_erase` will be removed soon by #5501 :)

Reply:
> Please update for other nodes in another PR. This PR is about RELU6.

I've removed the changes in other nodes.

> @jinevening FYI, shape_erase will be removed soon by #5501 :)

Thanks for sharing the plan.